k8s/namespaces/neuvector/fluentbit.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
missing the secret? https://github.com/hmcts/cnp-flux-config/blob/master/k8s/sandbox/common/neuvector/fluentbit-log.yaml